.github/workflows/cri_test.yml

```diff
@@ -68,3 +68,49 @@ jobs:
     - name: Cleaning
       if: ${{ always() }}
       run: ./scripts/github_runner/clean_cri_runner.sh
+  
+  gvisor-cri-tests:
+    name: gVisor CRI tests
+    env:
+        GITHUB_RUN_ID: ${{ github.run_id }}
+        GITHUB_VHIVE_ARGS: "-dbg"
+    runs-on: ubuntu-18.04
```

Comment:
since it's a fresh VM (not like with self-hosted), one has to set up it first, executing the same scripts as you do when running on a fresh node.